src/MuxPort.cpp

```diff
@@ -96,12 +96,11 @@ void MuxPort::handleBladeIpv4AddressUpdate(boost::asio::ip::address address)
 {
     MUXLOGDEBUG(boost::format("port: %s") % mMuxPortConfig.getPortName());
 
-    boost::asio::io_service &ioService = mStrand.context();
-    ioService.post(mStrand.wrap(boost::bind(
+    boost::asio::post(mStrand, boost::bind(
```

Comment:
`wrap()` occurs in other places too, do we plan to remove them all? If yes, do you want to put it in the same PR?

Reply:
This PR only removes the `wrap` in the `MuxPort` class.